### PR TITLE
Add bug reports

### PR DESCRIPTION
**Pull request by Gianfranco97**
Hi, @ajsb85 

In this PR add bug reports using Bugsnag.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,41 +2,45 @@
...
+        <!--<meta name="twitter:app:url:ipad" content="flyve://....">-->
+        <meta name="twitter:app:name:googleplay" content="Flyve MDM Agent">
+        <meta name="twitter:app:id:googleplay" content="org.flyve.mdm.agent">
+        <meta name="twitter:app:url:googleplay" content="market://details?id=org.flyve.mdm.agent">
 
-<!--<meta name="apple-itunes-app" content="app-id=#########, app-argument: flyve://....">-->
+        <!--<meta name="apple-itunes-app" content="app-id=#########, app-argument: flyve://....">-->
 
+        <script 
+            src="//d2wy8f7a9ursnm.cloudfront.net/bugsnag-3.min.js"
```
Comment (ajsb85) [inline index.html]: Could you use this file stored in the same branch?

Comment (ajsb85) [reply index.html]: @Gianfranco97 Any update about this request?

**Review by ajsb85 — CHANGES_REQUESTED** [review]

**Comment by ajsb85** [comment]
Hi, @Gianfranco97 

Could you use the library for bugsnag stored in our repo instead cloudfront?

Thanks.

**Comment by Gianfranco97** [comment]
@ajsb85 I have already made the requested changes, could you please review it?

**Review by ajsb85 — APPROVED** [review]
Reviewed

**Pull request merged**